tests/cases/functions/declaration

```diff
@@ -7,7 +7,7 @@ declare(strict_types=1);
 
 namespace Vendor;
 
-function foo  ( $bar  ,&$baz,  bool$qux, ? string  $quux    =null   ,  string...$quuz   ):
+function foo  ( $bar  ,&$baz,  bool$qux, ? string  $quux    =null   ,  string...  $quuz   ):
```

Comment:
I see, there isn't a dedicated fixture because these are already dedicated to whitespace